cmd/kubeadm/app/cmd/join.go

```diff
@@ -159,17 +153,12 @@ func NewCmdJoin(out io.Writer, joinOptions *joinOptions) *cobra.Command {
 		Long:  joinLongDescription,
 		Run: func(cmd *cobra.Command, args []string) {
 
-			c, err := joinRunner.InitData(args)
+			_, err := joinRunner.InitData(args)
 			kubeadmutil.CheckErr(err)
```

Comment:
these two lines are not needed:
```
_, err := joinRunner.InitData(args)
kubeadmutil.CheckErr(err)
```

Reply:
might be i am missing something but in updated PR i need
c, err := joinRunner.InitData(args)
kubeadmutil.CheckErr(err)
wdyt?

Reply:
```
_, err := joinRunner.InitData(args)
kubeadmutil.CheckErr(err)
```
^ these two lines are not needed.

as you can see:
```
err = joinRunner.Run(args)
```
will call InitData():
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/phases/workflow/runner.go#L193-L206